tests/backstop/backstop-settings.js

```diff
@@ -0,0 +1,121 @@
+/*
+  How to use
+
+  backstop reference --configPath=backstop-settings.js
+       backstop test --configPath=backstop-settings.js
+
+  backstop reference --configPath=backstop-settings.js --refhost=http://example.com
+       backstop test --configPath=backstop-settings.js --testhost=http://example.com
+
+  backstop reference --configPath=backstop-settings.js --paths=/,/contact
+       backstop test --configPath=backstop-settings.js --paths=/,/contact
+
+  backstop reference --configPath=backstop-settings.js --pathfile=paths
+       backstop test --configPath=backstop-settings.js --pathfile=paths
+
+ */
+
+/*
+  Set up some variables
+ */
+var arguments = require('minimist')(process.argv.slice(2)); // grabs the process arguments
+var defaultPaths = ['/']; // By default is just checks the homepage
+var scenarios = []; // The array that'll have the pages to test
+
+/*
+  Work out the environments that are being compared
+ */
+// The host to test
+if (!arguments.testhost) {
+  arguments.testhost  = "http://127.0.0.1:8000"; // Default test host
+}
+// The host to reference
+if (!arguments.refhost) {
+  arguments.refhost  = "http://127.0.0.1:8000"; // Default test host
+}
+/*
```

Comment:
@panshulK Since we've already added default refhost & testhost url, so this should work in IDE, local as well as in travis. So, instead of managing two files i.e `backstop.json` & this file i.e `backstop-settings.js` . Can't we remove backstop.json file and incorporate changes from that file here ? 

I know ticket mentions to use separate command  but I think we can avoid this.

cc: @chandan-singh7929